### ISSUE TITLE
Amazon Japan Support

### ISSUE THREAD

**Issue by jitaroo** [Hari-Nagarajan/nvidia-bot#301]
Is it possible to add amazon japan support?

**Comment by worst-mod-confirmed** [comment]
I don't see why it wouldnt be supported, have you tried it?

**Comment by jitaroo** [comment]
yes, ive tried it with --test and without. It doesnt even log in. I tried logging in as soon as the browser appeared however it didnt proceed to the page where it loads the asins.

**Comment by worst-mod-confirmed** [comment]
I added the signin and home page. See that lets you log in. We will need the title text from the checkout, and add to cart pages to add full functionallity. Give the "amazon-offer-pages" branch a try.

**Comment by jitaroo** [comment]
Unfortunately it still didnt log me in. I logged in manually and it still didnt add items to cart. I got an error page when it tried to load the ASINs
